leaderboardsgg/leaderboard-site
# [NO-ISSUE] updated all packages, fixed configs, updated documentation

## What

updated all packages, fixed configs, updated documentation

## Acceptance

### Steps for testing

  - Run the site and ensure the components aren't exploding
  - test suite is passing
